local_simulation_server.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Local simulation server."""
+
+import os
+import socket
+import subprocess
+import sys
+
+HOST = ''  # Any host can connect
+PORT = int(sys.argv[1])  # Port to listen on
```

Comment:
I would set the port by default to 2000 to match the default value set in `webots_tcp_client.py`.